tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27, py35, py36, py37-dev
+envlist = py35, py36, py37-dev
```

Comment:
I know it is not entirely relevant for the PR but while you are at could you change the py37-dev environment to py37?

Reply:
I tried, but apparently itʼnot supported by Travis yet. I havenʼt checked Tox, though.

Reply:
Tox does support it. I changed `tox.ini` and will keep my eyes on Travis to see when they drop the suffix, too.